components/dashboard/script.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+
+const APP_NAME = "Gitpod";
+
+const vars = {
+    REACT_APP_HOSTNAME : APP_NAME,
+    REACT_APP_DOCS_URL : "https://www.gitpod.io/docs/",
+    REACT_APP_SOCIAL_URL : "https://community.gitpod.io/",
+    REACT_APP_FAVICON_192 : "./favicon192.png",
+    REACT_APP_FAVICON_256 : "./favicon256.png",
+    REACT_APP_FAVICON_512 : "./favicon512.png",
+    REACT_APP_ICON_1 : "./icons/gitpod.svg",
+    REACT_APP_ICON_2 : "../icons/gitpod.svg",
```

Comment:
@satyamkumar-cn Add a comment in here on why we are doing it this way

Reply:
Since the images are used in different levels in the directory structure, to accurately point to the image we have two choices here :  
-- Either use single variable and manipulate it before using to make it point to right spot
-- Or we can use multiples variables which rightly point to the images wrt various levels in the directory structure